gem/rbi-central.gemspec

```diff
@@ -31,7 +31,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency("rubocop-sorbet", ">= 0.6.11")
   spec.add_dependency("sorbet-static-and-runtime", ">= 0.5.11922")
   spec.add_dependency("spoom", ">= 1.2.0")
-  spec.add_dependency("tapioca", ">= 0.9.2")
+  spec.add_dependency("tapioca", ">= 0.17.1")
```

Comment:
TODO: revert this.

I just did this as a way to force `bundle update tapioca` to pick up `0.17.1`.

Turns out `bundle update --pre tapioca` could achieve the same thing directly